components/fieldgroup/stories/template.js

```diff
@@ -22,8 +29,33 @@ export const Template = ({
   return html`
     <div class=${classMap({
       [rootClass]: true,
-      [`${rootClass}--size${size?.toUpperCase()}`]: typeof size !== "undefined",
+      [`${rootClass}--${labelPosition}label`]: typeof labelPosition !== "undefined",
+      [`${rootClass}--${layout}`]: typeof layout !== "undefined",
       ...customClasses.reduce((a, c) => ({ ...a, [c]: true }), {}),
-  })}></div>
+    })}>
+      ${FieldLabel({
+        ...globals,
+        size: "m",
+        label: "Field Group Label",
+        alignment: labelPosition === "side" ?? "right",
+      })}
```

Comment:
I am seeing on the docs site `spectrum-FieldLabel--right` applied to the field label which includes `padding-inline: 0 var(--mod-fieldlabel-side-padding-right, var(--spectrum-fieldlabel-side-padding-right));`

in the storybook when displayed with a side label this padding is not there and the label is touching the radio button. 
![Screenshot 2023-03-01 at 12 23 34 PM](https://user-images.githubusercontent.com/63808889/222243192-3c03e76d-801d-4fb0-885b-2329cda9f326.png)

Looks like something is not working correctly where we are adding the `right` on, class is displaying as `spectrum-FieldLabel--true`